src/Component/FieldTypes/FieldTypeInterface.php

```diff
@@ -22,11 +22,9 @@ interface FieldTypeInterface
      * Return a HTML representation of the $field using the given $data and
      * $options.
      *
-     * @param mixed $data
-     *
      * @return string
      */
-    public function render(Field $field, $data, array $options);
+    public function render(Field $field, mixed $data, array $options);
```

Comment:
Does this change also fits this PR and BC promise?

```suggestion
    public function render(Field $field, mixed $data, array $options): string;
```

```patch
---     * @return string
```

Reply:
This is a BC break since we are allowing any type and here it forces only string type, this can be done in v2 only.
I'm also creating a PR to set array shape on this same method and also clean the PHPStan baseline.